### ISSUE TITLE
script/server does not work if diaspora installed from a tarball

### ISSUE THREAD

**Issue by pravi** [diaspora/diaspora#6300]
This check leads to failure when installed from a tarball. 

```
# Check if detached head state
  git_branch_name="$(git symbolic-ref HEAD 2>/dev/null)"
  if [ -z "$git_branch_name" ];
  then
    warning "You are in detached HEAD state!"
  fi
```
##

**Comment by pravi** [comment]
Created a patch http://anonscm.debian.org/cgit/pkg-ruby-extras/diaspora.git/tree/debian/patches/043-dont-do-git-checks.patch

**Comment by jhass** [comment]
I think I'd prefer checking repo presence by checking the return code of `git status` if the `git` command is available. Otherwise looking forward to your PR.

**Comment by saveriomiroddi** [comment]
@jhass Hello Jonne! I've created a PR following the workflow you specified.

**Comment by SuperTux88** [comment]
Fixed with #7712
